app/User.php

```diff
@@ -59,4 +59,18 @@ public function activities()
     {
         return $this->hasMany(Activity::class);
     }
+
+    public function getLearnedWords($id)
```

Comment:
`$id` is not needed anymore since you can just query the Lessons by calling the relationship `$this->lessons`

Reply:
This is resolved in  bff069f